src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/ThrowsTaglet.java

```diff
@@ -110,8 +110,7 @@ public Content getAllBlockTagOutput(Element holder, TagletWriter writer) {
         Content result = writer.getOutputInstance();
         Set<String> alreadyDocumented = new HashSet<>();
         result.add(throwsTagsOutput(tagsMap, writer, alreadyDocumented, typeSubstitutions, true));
-        result.add(inheritThrowsDocumentation(holder,
-                thrownTypes, alreadyDocumented, typeSubstitutions, writer));
+        result.add(inheritThrowsDocumentation(holder, thrownTypes, alreadyDocumented, typeSubstitutions, writer));
         result.add(linkToUndocumentedDeclaredExceptions(thrownTypes, alreadyDocumented, writer));
         return result;
```

Comment:
Content.add returns `this`, so you could simplify this further to

    return writer.getOutletInstance()
        .add(throwsTagOutput....)
        .add(inheritThrowsDocumentation...)
        .add(linktoUndocumentedExceptions...);